### DISCUSSION TITLE
add dialog fot tree node

### DISCUSSION THREAD

**Discussion by qingtingzhiyu39** [eclipse-theia/theia#14493] [General]
hi,I have a question. At what point in the tree does a pop-up window appear with confirm and cancel buttons, and is located near the tree node? I want to ask if there is a native API like this?

**Comment by msujew** [comment]
Hey @qingtingzhiyu39,

I don't really follow. Dialogs are usually triggered on a click callback of a tree node but aren't really in any way associated with the tree itself. You can use the [Dialog API](https://github.com/eclipse-theia/theia/blob/master/packages/core/src/browser/dialogs.ts) to build dialogs.
